crypto/ui/ui_openssl.c

```diff
@@ -127,10 +127,6 @@
 #  define TTY_set(tty,data)      ioctl(tty,TIOCSETP,data)
 # endif
 
-# if !defined(_LIBC) && !defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_VMS) && ! (defined(OPENSSL_SYS_TANDEM) && defined(_SPT_MODEL_))
```

Comment:
I think this should stay, although the VMS bit could probably also go.

```
# if !defined(_LIBC) && !defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_VMS)
```

Reply:
I have a fixup commit that puts the ioctl.h include inside the termio and sgtty cases, and restores the conio include.  Please take a look.